### PR TITLE
Html homepage

### PR DESCRIPTION
**Pull request by Vigrouze**
Hey guys,
Could you please check my "Html homepage " code ? ;)
Thx

### CODE REVIEW

**Review by Joelebucheron — APPROVED** [review]
Let's goooooooo

File: app/assets/stylesheets/components/_cards_homepage.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  .cards {
+    grid-template-columns: 1fr 1fr;
+  }
+}
+
+@media(max-width: 768px) {
+  .cards {
+    grid-template-columns: 1fr;
+  }
+}
```
Comment (ndgdl) [inline app/assets/stylesheets/components/_cards_homepage.scss:30]: Might be better in the pages > home

File: app/assets/stylesheets/components/_cards_homepage.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.cards {
+   display: grid;
+   grid-template-columns: repeat(3, 1fr);
+   grid-gap: 16px;
```
Comment (ndgdl) [inline app/assets/stylesheets/components/_cards_homepage.scss:4]: That as well

**Review by ndgdl — COMMENTED** [review]
Just for better organisation of the files: move the layout related code to the pages > home (all the grid stuff)
Everything else is perfect!

**Pull request closed without merging**